packages/x-date-pickers/src/DesktopNextDateTimePicker/DesktopNextDateTimePicker.test.tsx

```diff
@@ -0,0 +1,311 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { act, fireEvent, screen, userEvent } from '@mui/monorepo/test/utils';
+import 'dayjs/locale/ru';
```

Comment:
@LukasTy we had the same import on the legacy picker but the tests passes without it
Probably an old localization test that was removed since.

Removed from both